react/features/base/participants/functions.ts

```diff
@@ -407,6 +408,29 @@ export function getParticipantDisplayName(stateful: IStateful, id: string): stri
     return defaultRemoteDisplayName ?? '';
 }
 
+/**
+ * Returns the source names of the screenshare sources in the conference.
+ *
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's {@code getState} function to be used to
+ * retrieve the state.
+ * @returns {string[]}
+ */
+export function getRemoteScreensharesBasedOnPresence(stateful: IStateful): string[] {
```

Comment:
Yes, there is another list of screenshare sources based on the tracks in redux. `getRemoteScreensharesBasedOnPresence` is needed only for creating the SS participant tile for adding the source info to the receiver constraints so that the bridge can signal the source and then start sending the media. Once the source signaling is received, the remote track will be added to redux and then `getScreenshareParticipantIds` will return the correct info.

Reply:
Ouch, things are getting complex... can you think of a good place to document that? Maybe right here in this very function jsdoc.